src/System.ValueTuple/src/System.ValueTuple.csproj

```diff
@@ -29,6 +29,9 @@
     <TargetingPackReference Include="System" />
     <TargetingPackReference Include="System.Core" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)' == ''">
+    <TargetingPackReference Include="System.Private.CoreLib" />
```

Comment:
For the facade it could go either way, it would just be a two hoop type-froward in the implementation.